config/zones/CA-NB.yaml

```diff
@@ -14,18 +14,53 @@ capacity:
   wind: 294
 contributors:
   - jarek
+  - VIKTORVAV99
+emissionFactors:
+  direct:
+    unknown:
+      - datetime: 2019-01-01
+        _url: https://open.canada.ca/data/en/dataset/5a6abd9d-d343-41ef-a525-7a1efb686300/resource/0c58139d-1811-466a-97ba-4c84b3fd83a2
+        source: assumes 37.54% nuclear, 22.41% hydro, 14.53% gas, 13.71% coal, 6.65% wind, 3.78% biomass and 1.38% oil
+        value: 163.5
+  lifecycle:
+    unknown:
+      - datetime: 2019-01-01
+        _url: https://open.canada.ca/data/en/dataset/5a6abd9d-d343-41ef-a525-7a1efb686300/resource/0c58139d-1811-466a-97ba-4c84b3fd83a2
+        source: assumes 37.54% nuclear, 22.41% hydro, 14.53% gas, 13.71% coal, 6.65% wind, 3.78% biomass and 1.38% oil
+        value: 211.9
 fallbackZoneMixes:
-  _source: Canada NEB yearly data for 2018
-  _url: https://www.neb-one.gc.ca/nrg/ntgrtd/mrkt/nrgsstmprfls/nb-eng.html
   powerOriginRatios:
-    value:
-      biomass: 0.03
-      coal: 0.18
-      gas: 0.12
-      hydro: 0.21
-      nuclear: 0.39
-      wind: 0.07
-flag_file_name: ca.png
```

Comment:
No longer used by the frontend so figured I'd do some cleanup while editing the file.